### ISSUE TITLE
Add term femoral neck with a collar-like bony outgrowth

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10374]
**Preferred term label:**
femoral neck with a bony collar

**Synonyms**
bony collar of femoral neck, femoral neck with a collar-like, bony outgrowth

**Definition (free text, please give PubMed ID)**
Femoral neck with bony protrusion (“collar”)
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/9d2aef9a-4d76-481d-8599-15a43b198b25)
See arrows, from PMID: 12624140

**Parent term (use hpo.jax.org/app)**
Abnormal femoral neck morphology HP:0003367

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROCAPITOFEMORAL DYSPLASIA; ACFD, OMIM607778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000775
- label: Femoral neck bony collar
- parents: [Abnormal femoral neck morphology (HP:0003367)](https://hpo.jax.org/app/browse/term/HP:0003367)
- pmid: PMID:12624140
- def: A bony protrusion surrounding the femoral neck and said to resemble a collar.
- comment: This feature has been observed in Acrocapitofemoral dysplasia; See Figure 4 of PMID:12624140.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10374
- synonyms:
